src/ui/FormWidgets/MultiSelect.ts

```diff
@@ -114,16 +114,16 @@ export class MultiSelect implements IFormWidget, IFormWidgetSettable {
 
   private buildContent() {
     this.element = <HTMLSelectElement>$$('select', {
-      className: 'mdc-multi-select mdl-list',
```

Comment:
I couldn't find any reference to `mdl-list` anywhere, I assume it was meant to be `mdc-list` and the component only had the correct look because of that typo.